src/txmempool.cpp

```diff
@@ -727,6 +728,9 @@ void CTxMemPool::check(const CCoinsViewCache *pcoins) const
         assert(it2 != mapTx.end());
         assert(&tx == it->second);
     }
+    for (const auto& entry : mapDeltas) {
+        assert(mapTx.count(entry.first) > 0);
```

Comment:
Deltas aren't necessarily in the pool (needed to prioritise ones that would be rejected otherwise).